.env.sample

```diff
@@ -2,6 +2,7 @@ NODE_ENV=
 
 RETRYABLE_MONITORING_SLACK_TOKEN=
 RETRYABLE_MONITORING_SLACK_CHANNEL=
+PRIVATE_KEY=
```

Comment:
Could we pls make it `RETRYABLE_MONITORING_PRIVATE_KEY` to allow for more control over future keys?